logic/dapp.js

```diff
@@ -43,17 +43,19 @@ __private.unconfirmedAscii = {};
  * @param {Object} network
  * @todo Add description for the params
  */
-// Constructor
```

Comment:
These `// Constructor` comments should be removed from the files where the function was already converted to a class.

Reply:
Removed.